autoarray/inversion/regularization/regularization_util.py

```diff
@@ -1,7 +1,61 @@
 import numpy as np
+from typing import Tuple
 
 from autoarray import numba_util
 
+from autofit import exc
+
+
+def reg_split_from(

```

Comment:
I've still not quite wrapped my head round what this code is doing.

Might be helpful if you try and expand on it in the docstring below.